pipelines/broad/arrays/imputation/Imputation.wdl

```diff
@@ -94,7 +95,19 @@ workflow ImputationPipeline {
       bcftools_docker = bcftools_docker_tag
   }
 
-  scatter (referencePanelContig in referencePanelContigs) {
+  scatter (contig in contigs) {
+
+    String reference_filename = reference_panel_path + "ALL.chr" + contig + ".phase3_integrated.20130502.genotypes.cleaned"
```

Comment:
I don't like this. It is not flexible in case someone wants to use alternate resources. Suggestions for alternative methods are welcomed!

Reply:
I think it's fine. I've seen worse.